### ISSUE TITLE
Consolidate our contact/sign-up vectors

### ISSUE THREAD

**Issue by afeld** [18F/cg-landing#66]
At the moment, the "Contact" link at the top of the homepage goes down to the form to "Sign up for updates", but I imagine what a lot of people want is actual contact info, which is available from the "cloud.gov help" link in the footer. Not sure of the ideal way to reconcile this, since I don't think we want people emailing us to say "can I get access?".

**Comment by brittag** [comment]
[Related user comment](https://news.ycombinator.com/item?id=11839634): "The "Contact us" section is actually a "Subscribe to our newsletter", I would look into changing the copy there and/or providing actual contact details."

**Comment by mogul** [comment]
The [Google Form that the "Sign up for updates" form encapsulates](https://docs.google.com/a/gsa.gov/forms/d/1S8P20K7yFTMAnpG9FT5r7aW5ja_phVcRqJbRlJqvqjM/edit) has been gathering e-mail addresses for people interested in cloud.gov since we first announced it... Currently [~950 e-mail addresses](https://docs.google.com/spreadsheets/d/1jZpQgOmdYb7MqYoas9POYzTee8-lsTSFT4lNxltCQ74/edit#gid=967877376). We have used a subset of .gov addresses from this list to generate some leads for user-testing via [another form](https://docs.google.com/a/gsa.gov/forms/d/1xpNaKNyn9cezF3dhazyI41pKNjICkA60ZZ2RlSRZykI/edit), and I thought that form was good at disambiguating motivations. In general, though, the big list is at bare minimum people who know want to know more about cloud.gov's availability... including many people outside government, eg press or vendors.

More motivated people looked around in the docs and found [the help page](https://docs.cloud.gov/help/), and mailed the Google Group behind cloud-gov-support@gsa.gov directly. Others were referred to that address when they talked directly to someone at 18F, eg in intake meetings. For a time that Google Group was flooded with business inquiries which don't really concern the whole team. So then we made cloud-gov-inquiries@gsa.gov which cut down on the noise of the former, but is not explicitly pointed to anywhere from outside, just for referrals.

In general, we need to unify and streamline all of this for users: 
- We want people to say up front what their interest in cloud.gov is! Then from that, and whether they have a .gov/.mil address, we can do some pre-qualification on what actions are available to them:
  - We want people who just want to know about availability to be notified when we blog/announce updates. 
  - We want .gov/.mil people also interested in being user-research candidates to go into a list that UX people can pull from. 
  - We want .gov/.mil people who want sandbox accounts to be able to get them <-- either manually as now, or @dlapiduz needs to say when we should open this up and be self-service, or if we should throttle it somehow, eg take a list and give out accounts at X per month.
  - We want people with existing accounts to be able to ask for support.
  - We want potential buyers who want to get sandbox accounts or buy service to be able to get in a queue for bizdev activity.

I don't know what to do with this... It looks like a bunch of little stories that already exist, but without regard to coherence and cleanly meeting needs for users and for us. Does it need UX research? Just some design?

**Comment by nikzei** [comment]
tl;dr is that we need to update our "sign up for updates" so that it captures more meaningful information + provides our contact info. Let's revisit. Moving this to backlog.

**Comment by berndverst** [comment]
This issue seems to encapsulate a lot of different things to be improved.

The sign up form was updated here:
https://github.com/18F/cg-landing/pull/99

I suppose what remains to be done here is just change our copy to make sure this is a signup list to get more information. We should make our direct email address more easily discoverable.

**Comment by nikzei** [comment]
See 18F/cg-product#346.

**Comment by nikzei** [comment]
We've updated mailchimp to gather more meaningful info from visitors.
We've noted that we need to change the contact page so that we present ourselves as selling something rather than gathering email addresses for a newsletter (!). Noted in 18F/cg-product#346 . I'm closing this.
